File: EJERCICIOS/3.py
#Escribir un algoritmo que calcule el area de un triangulo
def area(base, altura):
    return (base*altura)/2

print("Ingrese la base del tringulo")
b = float(input())
print("Ingrese la altura del tringulo")
a = float(input())
print(area(b,a))

File: EJERCICIOS/typescript/3.ts
import readline from 'node:readline/promises';
import { stdin as input, stdout as output } from 'node:process';

const rl = readline.createInterface({ input, output });

function area(base: number, altura: number): number {
  return (base * altura) / 2;
}

console.log('Ingrese la base del triangulo');
const base = Number(await rl.question(''));

console.log('Ingrese la altura del triangulo');
const altura = Number(await rl.question(''));

console.log(area(base, altura));
rl.close();

